build.gradle

```
@@ -437,23 +430,13 @@ tasks.register("downloadFilebeat") {
 
     doLast {
         download {
-            String beatVersion = project.ext.get("stackArtifactSuffix")
+            String beatVersion = project.ext.get("artifactApiVersion")
```

```suggestion
            String beatsVersion = project.ext.get("artifactApiVersion")
```

The project is named beats, to be simetric with `beatsArchitecture`